metrics/xrootd_metrics.go

```diff
@@ -438,6 +444,36 @@ var (
 		Help: "Number of scheduler threads",
 	}, []string{"state"})
 
+	ThreadCreations = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_creations",
+		Help: "Number of scheduler thread creations",
+	})
+
+	ThreadDestructions = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_destructions",
+		Help: "Number of scheduler thread destructions",
+	})
+
+	ThreadLimitReached = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "xrootd_sched_thread_limit_reached",
+		Help: "Number of scheduler thread limit reached",
+	})
+
+	Jobs = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "xrootd_sched_jobs",
+		Help: "Number of scheduler jobs requiring a thread",
+	})
+
+	LongestQueue = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "xrootd_sched_longest_queue",
```

Comment:
It feels like you have a pretty coherent naming taxonomy, except for this metric because of "longest_queue". In this name, "longest" describes the queue but is placed before it in the name. How would you feel about `xrootd_sched_queue_longest` or something like that?